releases/v0.8.0.md

```diff
@@ -0,0 +1,67 @@
+# Release Notes for v0.8.0
+
+Release Date: Oct TBD, 2023
```

Comment:
Update date

Reply:
We don't know the release date yet. I am thinking we can update in a subsequent PR next week.

Reply:
I guess it definitely won't be in October tho

Reply:
Yeah, sorry, I saw your note about updating the date after I posted this.